operator/config/samples/component-integration-installed/operator_v1alpha1_moduletemplate_istio_stable.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: operator.kyma-project.io/v1alpha1
+kind: ModuleTemplate
+metadata:
+  name: moduletemplate-sample-istio
+  namespace: default
+  labels:
+    "operator.kyma-project.io/managed-by": "kyma-operator"
+    "operator.kyma-project.io/controller-name": "istio"
+    "operator.kyma-project.io/channel": "stable"
+spec:
+  data:
+    istio: |-
+      kind: IstioConfiguration
+      group: kyma.kyma-project.io
+      resource: istioconfigurations
+      version: v1alpha1
+      spec:
+        foo: "Stable Speed!"
```

Comment:
should be revised, without component name + spec inside spec
**-- same for other moduletemplate resources**